sdk/core/azure-core/src/main/java/com/azure/android/core/http/interceptor/LogUtils.java

```diff
@@ -40,33 +30,45 @@ static long getContentLength(Headers headers) {
      * @return "Log body" if the body should be logged in its entirety, otherwise a message indicating why the body
```

Comment:
Definitely need to get rid of this magic string. Even if we made this into a constant, it's still a little suspicious. Why not name this method something like `getBodySummary()` and then if it returns `null` log the body text, otherwise log the summary?